src/views/mixed-content/government/top-countries-with-more-websites-with-mixed-content-adjusted/index.js

```diff
@@ -0,0 +1,165 @@
+import * as features from '../features.json';
+import {api} from '../../../../client';
```

Comment:
I made a special 'hack' to avoid this `../../../../../` situations. The hack is to add to the environment the path of the source folder. So now, you can access everything under `src` directly. For example, replace this line with:

```
import {api} from 'client';
```

And it will work

Reply:
Done!